store/mongo/datastore_mongo.go

```diff
@@ -209,11 +215,11 @@ func (db *DataStoreMongo) GetDevices(ctx context.Context, q store.ListQuery) ([]
 		"$facet": bson.M{
 			"results": []bson.M{
 				sortQuery,
-				bson.M{"$skip": q.Skip},
+				{"$skip": q.Skip},
```

Comment:
is there a reason for that change?

Reply:
This is just a side-effect of running gofmt with the `-s` flag ("simplify").